src/java_bytecode/java_bytecode_language.h

```diff
@@ -48,6 +48,7 @@ Author: Daniel Kroening, kroening@kroening.com
   "                                  A '.*' wildcard is allowed to specify all class members\n"
 
 #define MAX_NONDET_ARRAY_LENGTH_DEFAULT 5
+#define MAX_NONDET_STRING_LENGTH 20
```

Comment:
Is that the value by default when no `--string-max-input-length` is given?

Reply:
Yes, it is.

Reply:
It's not usual to have a limit by default in the cbmc options. This kind of option is generally meant to restrict the search space, not to extend it. For instance, if we run a bmc that never ends, then we retry with `-unwind 100`, knowing that we are restricting the model, and thus eliminating possible assertion failures. If omitting the `--unwind` option cbmc would fall back to a default unwinding value, it would be very confusing and the user might think that his program is valid although some assertion violations are accessible at higher unwindings.

I believe it should be the same reasoning in that case. What do you think?